stdlib/Dates/src/adjusters.jl

```diff
@@ -63,8 +61,6 @@ julia> Dates.lastdayofweek(DateTime("1996-01-05T12:30:00"))
 1996-01-07T00:00:00
 ```
 """
-function lastdayofweek end
```

Comment:
Removing these will attach the docstrings to a specific method instead of the function